src/sentry/conf/server.py

```diff
@@ -740,9 +740,6 @@ def create_partitioned_queues(name):
 # The default value for project-level quotas
 SENTRY_DEFAULT_MAX_EVENTS_PER_MINUTE = '90%'
 
-# The maximum number of events per minute the system should accept.
-SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE = 0
```

Comment:
Do you want to add this into the `config.yml` now? Or just not surface this at all anymore?